interp/cfg.go

```diff
@@ -211,25 +211,22 @@ func (interp *Interpreter) cfg(root *node, importPath string) ([]*node, error) {
 
 		case labeledStmt:
 			label := n.child[0].ident
-			if sym, _, ok := sc.lookup(label); ok {
-				if sym.kind != labelSym {
-					err = n.child[0].cfgErrorf("label %s not defined", label)
-					break
-				}
```

Comment:
It seems to me that this check (that the symbol we found is indeed a label kind), is orthogonal to the actual change (of lookup becoming lookdown). Do you confirm that you removed this check because it was actually never needed?

Reply:
This check can be removed.

I just found  that goto labels can not be stored in scope symbols, but need a serparate namespace. In any case the current implementation is flawed, and does not support collisions between goto labels and other symbols, as shown by the following:

```go
package main

func main() {
    here := 0
    println("before goto")
here:
    if here > 0 {
        return
    }
    here++
    println("ok", here)
    goto here
}
```

Expected:
```console
$ go run ./sample.go
before goto
ok 1
```

Got:
```console
$ yaegi ./sample.go
./sample.go:6:1 label here not defined
```

So I propose to merge this PR as is, as it is better than current situtation, and open a new issue with the above example regarding label/symbol collision, which will require a deeper change in the way the labels are stored.